inc/Engine/Optimization/DelayJS/HTML.php

```diff
@@ -230,6 +230,8 @@ public function replace_scripts( $matches ): string {
 			}
 		}
 
+		$delay_js = str_replace( 'src=', 'data-rocket-src=', $delay_js );
```

Comment:
This needs to be changed a bit, as this could have unexpected matching, if there is any other attribute that ends with `src`.